.github/workflows/test.yml

```diff
@@ -12,7 +12,7 @@ jobs:
           fetch-depth: 1
```

Comment:
Could you split this commit into another PR?

Reply:
I omitted the commit